apps/vaporgui/VDoubleLineEdit.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <iostream>
+
+#include <string>
+#include <sstream>
+#include <iomanip>
+
+#include <QString>
+
+#include "VNumericLineEdit.h"
+
+//! \class VDoubleLineEdit
+//! \ingroup Public_GUI
+//! \brief A wrapper for a QLineEdit that handles user input of type double,
+//! and provides Vapor's standard setters, getters, and signals
+
+class VDoubleLineEdit : public VNumericLineEdit {
+    Q_OBJECT
+
+    public:
+        VDoubleLineEdit( double value, bool useMenu=true );
```

Comment:
Feel free to reject this but does the constructor have an initial value? It will always be replaced by the update logic once the widget is shown to the user. Qt widgets do not have initial values in the constructor for I am assuming the same reason.

Reply:
I debated this too.  In some cases it's useful to provide an initial value, such as for setting up the DVR's Volume Density, or the SliceRenderer's Quality widgets.  These don't get changed on Update.  But you're right that in other cases, an initial value just gets overwritten during Update.  I think that just using an optional default value satisfies both requirements.